### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1307] [General]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/e3e905a6-78d9-4139-a03f-24970bb45b3a)
When launching an app, the app is overlapped by waybar as shown in top.
How to prevent this

**Comment by kRHYME7** [comment]
is the browser running on wayland? 

What happen if you toogle float it . ``` ALT D ```

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/09e57547-a35f-4386-b883-bc2eb0d59bff)

**Reply by rubiin** [reply]
if i press mod+w , it does what i expect it to look like when it first opened
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/3bebd1c6-4dd1-4a9d-b8bb-87f28bf8c544)

**Reply by kRHYME7** [reply]
do Win + w  toggles floating?

**Reply by rubiin** [reply]
yes it does

**Reply by kRHYME7** [reply]
firefox? is your browser running on wayland? ``` hyprctl clients ```  (This looks an older common issue)

**Comment by dieBakterie** [comment]
![grafik](https://github.com/prasanthrangan/hyprdots/assets/50206827/1fd5bf66-ed19-4b58-9545-e221ce726fd7)
With firefox it happens for me and with other apps like kitty too bot its only a little bit

**Comment by Enigma1309** [comment]
try `hyprctl clients `
get the window class
add it to your userprefs.conf
![image](https://github.com/prasanthrangan/hyprdots/assets/94917588/4ac5c425-9a23-47b5-84cf-d185a94b5507)
(for example- brave)

**Reply by rubiin** [reply]
can this also be added on `userperf`

**Reply by Enigma1309** [reply]
what's that? by userprefs.conf i mean 
`~/.config/hypr/userprefs.conf`

**Reply by rubiin** [reply]
yup that file
